frontend/src/reducers/mineReducer.js

```diff
@@ -43,6 +44,11 @@ const mineReducer = (state = initialState, action) => {
         ...state,
         mineBasicInfoList: action.payload,
       };
+    case actionTypes.STORE_MINE_DOCUMENTS:
+      return {
+        ...state,
+        mineDocuments: action.payload.mine_documents,
```

Comment:
Curious if we're now duplicating documents in our store, once as child properties of the tailings reports and here? Should we refactor the tailings screen after this change to map based on a relationship, or is this ok?